File: src/js/personas.js
export class Personas {
    tipoId;
    numId;
    nombre;
    apellido;
    correo;
    numeroTelefono;
    sexo;
    direccion;
    edad;
    constructor(
        tipoId,
        numId,
        nombre,
        apellido,
        correo,
        numeroTelefono,
        sexo,
        direccion,
        edad) {
            this.tipoId = tipoId;
            this.numId = numId;
            this.nombre = nombre;
            this.apellido = apellido;
            this.correo = correo;
            this.numeroTelefono = numeroTelefono;
            this.sexo = sexo;
            this.direccion = direccion;
            this.edad = edad;
        }

    get getTipoId() {
        return this.tipoId;
    }

    set setTipoId(newTipoId) {
        this.tipoId = newTipoId;
    }

    get getNumId() {
        return this.numId;
    }

    set setNumId(newNumId) {
        this.numId = newNumId;
    }

    get getNombre() {
        return this.nombre;
    }

    set setNombre(newNombre) {
        this.nombre = newNombre;
    }

    get getApellido() {
        return this.apellido;
    } 

    set setApellido(newApellido) {
        this.apellido = newApellido;
    }

    get getCorreo() {
        return this.correo;
    }

    set setCorreo(newCorreo) {
        this.correo = newCorreo;
    }

    get getNumeroTelefono() {
        return this.numeroTelefono;
    }

    set setNumeroTelefono(newNumeroTelefono) {
        this.numeroTelefono = newNumeroTelefono;
    }

    get getSexo() {
        return this.sexo;
    }

    set setSexo(newSexo) {
        this.sexo = newSexo;
    }

    get getDireccion() {
        return this.direccion;
    }

    set setDireccion(newDireccion) {
        this.direccion = newDireccion;
    }

    get getEdad() {
        return this.edad;
    }

    set setEdad(newEdad) {
        this.edad = newEdad;
    }
}

File: Personas.java
public class Personas {
    private int tipoId;
    private int numId;
    private String nombre;
    private String apellido;
    private String correo;
    private int numeroTelefono;
    private int sexo;
    private String direccion;
    private int edad;

    // Getter
    public int getTipoId() {
        return tipoId;
    }

    public int getNumId() {
        return numId;
    }

    public String getNombre() {
        return nombre;
    }

    public String getApellid() {
        return apellido;
    }

    public String getCorreo() {
        return correo;
    }

    public int getNumeroTelefono() {
        return numeroTelefono;
    }

    public int getSexo() {
        return sexo;
    }

    public String getDireccion() {
        return direccion;
    }

    public int getEdad() {
        return edad;
    }

    // Setter
    public void setTipoId(int newTipoId) {
        this.tipoId = newTipoId;
    }

    public void setNumId(int newNumId) {
        this.numId = newNumId;
    }

    public void setNombre(String newNombre) {
        this.nombre = newNombre;
    }

    public void setApellid(String newApellido) {
        this.apellido = newApellido;
    }

    public void setCorreo(String newCorreo) {
        this.correo = newCorreo;
    }

    public void setNumeroTelefono(int newNumeroTelefono) {
        this.numeroTelefono = newNumeroTelefono;
    }

    public void setSexo(int newSexo) {
        this.sexo = newSexo;
    }

    public void setDireccion(String newDireccion) {
        this.direccion = newDireccion;
    }

    public void setEdad(int newEdad) {
        this.edad = newEdad;
    }
}
